week-3/introduction_to_java_book_tasks/_9_4_using_random/src/main/java/job_hacker/tasks/Main.java

```diff
@@ -0,0 +1,17 @@
+package job_hacker.tasks;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        Random rand = new Random(1000);
+        for (int i = 0;; i++){
```

Comment:
why made this please use for loop properly

Reply:
I meant to use it like that to stop the last comma and not for vacuum.
I don't have another better solution.